src/store/reducers/index.js

```diff
@@ -30,4 +30,4 @@ export default combineReducers({
     [Target.PUBLISHER]  : publishersReducer,
     [Target.BRANCH]     : branchesReducer,
     [Target.BORROWER]   : borrowersReducer,
```

Comment:
other reducers need to be commented out. It is throwing error on console. 

export default combineReducers({
  [Target.BOOK]: booksReducer
});

Reply:
Yeah, it's a warning, but should be gone once other reducers are filled in